### ISSUE TITLE
Show more grades of conformance

### ISSUE THREAD

**Issue by jenoe** [w3c/silver#303]
The new conformance system is partly based on a calculated scoring level plus a system of critical errors.

Passing the bronze level is an easy to communicate label to show the current state of accessibility.

## Problem

The new conformance system gives no clue about the progress over time and a more meaningful information about the actual level reached so far.

A website that has failed to reach bronze only by a tiny margin might want to communicate this internally as well as to the public. It can make a big difference for them compared to others that are still struggling with critical errors and a low overall score.

## Proposal

Provide an additional optional conformance information that communicates total score ("_A11Y Score x %_") and number of critical errors passed ("_6/7 critical tests passed_") as well. This could also be extended by a visualisation (eg like energy labels or gauge charts).

Such additional information could be an **incentive** for an organisation **to improve over time and communicate this progress**.

**Comment by lauracarlson** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by fstrr** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
